test/OpenTelemetry.Instrumentation.StackExchangeRedis.Tests/StackExchangeRedisCallsInstrumentationTests.cs

```diff
@@ -120,31 +120,31 @@ private static void VerifyActivityData(Activity activity, bool isSet, EndPoint e
-                Assert.Equal("SETEX", activity.Tags.FirstOrDefault(t => t.Key == SemanticConventions.AttributeDbStatement).Value);
+                Assert.Equal("SETEX", activity.TagObjects.FirstOrDefault(t => t.Key == SemanticConventions.AttributeDbStatement).Value);
             }
             else
             {
                 Assert.Equal("GET", activity.DisplayName);
-                Assert.Equal("GET", activity.Tags.FirstOrDefault(t => t.Key == SemanticConventions.AttributeDbStatement).Value);
+                Assert.Equal("GET", activity.TagObjects.FirstOrDefault(t => t.Key == SemanticConventions.AttributeDbStatement).Value);
             }
 
-            Assert.Equal(SpanHelper.GetCachedCanonicalCodeString(StatusCanonicalCode.Ok), activity.Tags.FirstOrDefault(t => t.Key == SpanAttributeConstants.StatusCodeKey).Value);
-            Assert.Equal("redis", activity.Tags.FirstOrDefault(t => t.Key == SemanticConventions.AttributeDbSystem).Value);
-            Assert.Equal("0", activity.Tags.FirstOrDefault(t => t.Key == StackExchangeRedisCallsInstrumentation.RedisDatabaseIndexKeyName).Value);
+            Assert.Equal(SpanHelper.GetCachedCanonicalCodeString(StatusCanonicalCode.Ok), activity.TagObjects.FirstOrDefault(t => t.Key == SpanAttributeConstants.StatusCodeKey).Value);
+            Assert.Equal("redis", activity.TagObjects.FirstOrDefault(t => t.Key == SemanticConventions.AttributeDbSystem).Value);
+            Assert.Equal("0", activity.TagObjects.FirstOrDefault(t => t.Key == StackExchangeRedisCallsInstrumentation.RedisDatabaseIndexKeyName).Value);
 
             if (endPoint is IPEndPoint ipEndPoint)
             {
-                Assert.Equal(ipEndPoint.Address.ToString(), activity.Tags.FirstOrDefault(t => t.Key == SemanticConventions.AttributeNetPeerIp).Value);
-                Assert.Equal(ipEndPoint.Port.ToString(), activity.Tags.FirstOrDefault(t => t.Key == SemanticConventions.AttributeNetPeerPort).Value);
+                Assert.Equal(ipEndPoint.Address.ToString(), activity.TagObjects.FirstOrDefault(t => t.Key == SemanticConventions.AttributeNetPeerIp).Value);
+                Assert.Equal(ipEndPoint.Port.ToString(), activity.TagObjects.FirstOrDefault(t => t.Key == SemanticConventions.AttributeNetPeerPort).Value);
             }
             else if (endPoint is DnsEndPoint dnsEndPoint)
             {
-                Assert.Equal(dnsEndPoint.Host, activity.Tags.FirstOrDefault(t => t.Key == SemanticConventions.AttributeNetPeerName).Value);
-                Assert.Equal(dnsEndPoint.Port.ToString(), activity.Tags.FirstOrDefault(t => t.Key == SemanticConventions.AttributeNetPeerPort).Value);
+                Assert.Equal(dnsEndPoint.Host, activity.TagObjects.FirstOrDefault(t => t.Key == SemanticConventions.AttributeNetPeerName).Value);
+                Assert.Equal(dnsEndPoint.Port.ToString(), activity.TagObjects.FirstOrDefault(t => t.Key == SemanticConventions.AttributeNetPeerPort).Value);
```

Comment:
This should be an integer according to the [spec](https://github.com/open-telemetry/opentelemetry-specification/blob/b70565d5a8a13d26c91fb692879dc874d22c3ac8/specification/trace/semantic_conventions/span-general.md#general-network-connection-attributes). @cijothomas?

Reply:
I think we can change. let me push and test.

Reply:
just updated some places. still have to look very carefully if we missed something.